newproject2.php

```diff
@@ -21,6 +33,7 @@
     <img src="34DD0CEF-7DEB-4FAC-A0D5-F3E1E85C7082.jpg" class="images" alt="The Doctor (David Tennant) stares into the camera in a blue suit and against an orange background.">
```

Comment:
Excellent alt descriptions.

Reply:
Just because people are blind doesn't mean they should be bored right ;P